gallery.html

```diff
@@ -190,9 +190,8 @@ <h4>Dragon demo</h4>
                 <tr height="30px" />
 
                 <tr>
-                  
                   <td >
-                    <h4>Morse persistence demo</h4>
+                    <h4><a id="DragonDemo">Dragon demo</a></h4>
```

Comment:
as you can read, the list "features in action" includes the morse-smale complex, which is not in the dragon demo, but in the next one (morsePersistence).
thanks for fixing that.